sw/device/silicon_creator/lib/error.h

```diff
@@ -145,7 +147,12 @@ enum module_ {
   \
   X(kErrorRndBadCrc32,                ERROR_(1, KModuleRnd, kInvalidArgument)), \
   \
-  X(kErrorBootSvcBadHeader,           ERROR_(1, kModuleBootSvc, kInternal))
+  X(kErrorBootSvcBadHeader,           ERROR_(1, kModuleBootSvc, kInternal)), \
+  \
+  X(kErrorRomExtBootFailed,           ERROR_(1, kModuleRomExt, kFailedPrecondition)), \
+  \
+  /* The high-byte of kErrorInterrupt is modified with the interrupt cause */ \
+  X(kErrorRomExtInterrupt,            ERROR_(0, kModuleRomExtInterrupt, kUnknown)),
```

Comment:
Looks like an extra comma snuck in on line 155 :)

Reply:
Ah, you're right. Thanks! Fixed.